front/lib/api/assistant/conversation.ts

```diff
@@ -1655,3 +1669,39 @@ async function* streamRunAgentEvents(
     }
   }
 }
+
+async function isMessagesLimitReached({
+  owner,
+  plan,
+}: {
+  owner: WorkspaceType;
+  plan: PlanType;
+}): Promise<boolean> {
+  if (plan.limits.assistant.maxMessages === -1) {
+    return false;
+  }
+  /** Counting on workspace because user can be null */
+  const count = await Conversation.count({
```

Comment:
This is not good > we want count on AgentMessagge

Reply:
cf my com to stan

> Inner join returns all valid distinct couples (conversation, message)
I tested it before submitting the PR so it does what is expected and what we want